src/Sparrow/Json/AbstractBlittableJsonTextWriter.cs

```diff
@@ -695,6 +695,7 @@ protected void DisposeInternal()
             try
             {
                 FlushInternal();
+                _stream.Flush(); // flush the underlying stream as well
```

Comment:
why this was added? this is called during the dispose so I think dispose of that stream will do the flash? have you encountered an issue?

Reply:
TBH, since I've did manual testing and had multiple issues it is hard to tell now.
But it is missing, since I couldn't find it anywhere... just note that for `MemoryStream` that make no difference, since there is no where to flush to, but for other streams we must call the `Flush`.

I did those stream changes in a different commit so we can easy revert those if needed